.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
   - gem --version
   - bundle -v
 script:
+  - 'bundle exec puppet --version'
```

Comment:
Could you remove it please and add a PR/issue to modulesync_config instead? I'm not sure if we really need this, but it will be purged with the next sync. travis lists the installed gems, there is also the puppet version: https://travis-ci.org/voxpupuli/puppet-hiera/jobs/164738495#L488